firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -248,7 +252,6 @@ public void updateToLatestRelease_whenNewAabReleaseAvailable_showsUpdateDialog()
     firebaseAppDistribution.onActivityResumed(activity);
 
     // Update flow
-    verify(mockTesterSignInClient, times(1)).signInTester(activity);
```

Comment:
Removed this because with new signin checks, this will either be 0 if already signed in and 2 if not, because it will be triggered for both the updateToLatestRelease check and the checkForUpdate that is called within updateToLatestRelease in the context of testing.

Reply:
Hmmm. We should be able to reset the mock to 0 (if that doesn't happen automatically) And it should still be called at most one time (checkforupdate should not call it if the tester is already signed in)

It's possible to have the mock storage client return false the first time and then true the second. Let me know if you'd like to discuss in person :)